osc_sdk/sdk.py

```diff
@@ -370,7 +376,7 @@ def make_request(self, call, *args, **kwargs):
 
         request_url = "{}://{}".format(self.protocol, self.host)
 
-        self.response = self.get_response(
+        self.response = self.(
```

Comment:
Il faudrait remettre notre petit get_response ici :)

Reply:
thx for notice. modified to original.